docs/guides/options.md

```diff
@@ -1,138 +1,312 @@
-Options
-=======
+# Video.js Options Reference
 
-Setting Options
----------------
+> **Note:** This document is only a reference for available options. To learn about passing options to Video.js, see [the setup guide](setup.md#options).
 
-The Video.js embed code is simply an HTML5 video tag, so for many of the options you can use the standard tag attributes to set the options.
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-```html
-<video controls autoplay preload="auto" ...>
-```
 
-Alternatively, you can use the data-setup attribute to provide options in the [JSON](http://json.org/example.html) format. This is also how you would set options that aren't standard to the video tag.
+- [Standard `<video>` Element Options](#standard-video-element-options)
+  - [`autoplay`](#autoplay)
+  - [`controls`](#controls)
+  - [`height`](#height)
+  - [`loop`](#loop)
+  - [`muted`](#muted)
+  - [`poster`](#poster)
+  - [`preload`](#preload)
+  - [`src`](#src)
+  - [`width`](#width)
+- [Video.js-specific Options](#videojs-specific-options)
+  - [`aspectRatio`](#aspectratio)
+  - [`children`](#children)
+  - [`fluid`](#fluid)
+  - [`inactivityTimeout`](#inactivitytimeout)
+  - [`language`](#language)
+  - [`languages`](#languages)
+  - [`notSupportedMessage`](#notsupportedmessage)
+  - [`plugins`](#plugins)
+  - [`sourceOrder`](#sourceorder)
+  - [`sources`](#sources)
```

Comment:
Do we want to specify anything from the [default options](https://github.com/videojs/video.js/blob/master/src/js/player.js#L2752)? (Seems like some of these may not even exist anymore...)